tests/test_api.py

```diff
@@ -150,7 +150,7 @@ def generate_fizzbuzz_events(self) -> None:
     def redis_db_size(self) -> int:
         # dbsize could be an integer for a single node cluster or a dictionary
         # with one key value pair per node for a multi node cluster
-        dbsize = redis_client.dbsize()
+        dbsize: int = redis_client.dbsize()
```

Comment:
What's the `isinstance(dbsize, dict)` on the line below for? If this is an int why would that be needed?

Reply:
If the comment above was right then should the type be `int | dict`